src/main/webapp/resources/js/pages/projects/create/CreateProjectDetails.jsx

```diff
@@ -49,6 +50,11 @@ export function CreateProjectDetails({ form }) {
             pattern: /^[a-zA-Z0-9\s_-]+$/,
             message: i18n("CreateProjectDetails.name-characters"),
           },
+          {
+            type: "string",
+            min: 4,
```

Comment:
Project name actually needs to be **5** characters long.

Reply:
Updated and tests have been added for name required, length, and invalid characters